packages/firestore/src/remote/remote_event.ts

```diff
@@ -49,6 +56,30 @@ export class RemoteEvent {
      */
     readonly resolvedLimboDocuments: DocumentKeySet
   ) {}
+
+  /**
+   * Creates a synthesized RemoteEvent that can be used to apply a CURRENT
+   * status change for a query that is executed by a different tab.
```

Comment:
I would maybe expand this comment and call out that this is still a bit hacky.  Maybe:

HACK: Views require RemoteEvents in order to determine whether the view is CURRENT, but secondary tabs don't receive remote events. So this method is used to create a synthesized RemoteEvent that can be used to apply a CURRENT status change to a View, for queries executed in a different tab.

Reply:
Done (here and below)